src/Element/PluginSelect.php

```diff
@@ -127,6 +127,10 @@ public static function processPluginSelect(&$element, FormStateInterface $form_s
       $plugin = $plugin_manager->createInstance($target_plugin_id, $values['target_plugin_configuration']);
       if ($plugin instanceof PluginFormInterface) {
         $element['target_plugin_configuration'] = $plugin->buildConfigurationForm($element['target_plugin_configuration'], $form_state);
```

Comment:
okay, there's other issues about fixing the element and other items, which this might be part of